pkg/crc/preflight/preflight.go

```diff
@@ -128,6 +135,18 @@ func doFixPreflightChecks(checks []PreflightCheck) {
 	}
 }
 
+func doCleanUpPreflightChecks(checks []PreflightCheck) {
+	for _, check := range checks {
```

Comment:
iterating in reverse order would be safer to ensure we undo the changes in the right order.